Upstream: Not applicable, they use a different, albeit broken, system.
Summary: Respect DESTDIR

diff --git a/install.sh b/install.sh
index 401cbb4..eef2f32 100755
--- a/install.sh
+++ b/install.sh
@@ -1,22 +1,12 @@
 #!/bin/bash
 
 SRC_DIR=$(cd $(dirname $0) && pwd)
-ROOT_UID=0
-
-# Destination directory
-if [ "$UID" -eq "$ROOT_UID" ]; then
-  AURORAE_DIR="/usr/share/aurorae/themes"
-  SCHEMES_DIR="/usr/share/color-schemes"
-  PLASMA_DIR="/usr/share/plasma/desktoptheme"
-  LOOKFEEL_DIR="/usr/share/plasma/look-and-feel"
-  KVANTUM_DIR="/usr/share/Kvantum"
-else
-  AURORAE_DIR="$HOME/.local/share/aurorae/themes"
-  SCHEMES_DIR="$HOME/.local/share/color-schemes"
-  PLASMA_DIR="$HOME/.local/share/plasma/desktoptheme"
-  LOOKFEEL_DIR="$HOME/.local/share/plasma/look-and-feel"
-  KVANTUM_DIR="$HOME/.config/Kvantum"
-fi
+
+AURORAE_DIR="$DESTDIR/usr/share/aurorae/themes"
+SCHEMES_DIR="$DESTDIR/usr/share/color-schemes"
+PLASMA_DIR="$DESTDIR/usr/share/plasma/desktoptheme"
+LOOKFEEL_DIR="$DESTDIR/usr/share/plasma/look-and-feel"
+KVANTUM_DIR="$DESTDIR/usr/share/Kvantum"
 
 [[ ! -d ${AURORAE_DIR} ]] && mkdir -p ${AURORAE_DIR}
 [[ ! -d ${SCHEMES_DIR} ]] && mkdir -p ${SCHEMES_DIR}
